src/gleam/iterator.gleam

```diff
@@ -712,6 +712,50 @@ pub fn find(
   |> do_find(is_desired)
 }
 
+fn do_find_map(
+  continuation: fn() -> Action(a),
+  f: fn(a) -> Result(b, c),
+) -> Result(b, Nil) {
+  case continuation() {
+    Stop -> Error(Nil)
+    Continue(e, next) ->
+      case f(e) {
+        Ok(e) -> Ok(e)
+        Error(_) -> do_find_map(next, f)
+      }
+  }
+}
+
+/// Finds the first element in a given iterator for which the given function returns Ok(new_value),
+/// then returns the wrapped new_value.
+///
+/// Returns Error(Nil) if no such element is found.
```

Comment:
Could you wrap the inline code with backticks and wrap before 80 characters please

Reply:
:heavy_check_mark: